recipe/meta.yaml

```diff
@@ -228,93 +283,71 @@ outputs:
-    test:
-      imports:
-        - PyQt5.QtChart
-
-  - name: pyqt
-    build:
-      run_exports:
+        - {{ pin_compatible("qt-main", max_pin="x.x") }}
         - {{ pin_subpackage('pyqt', max_pin='x.x') }}
-    requirements:
-      run_constrained:                                            # [osx]
-        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
-      host:
-        - python
-      run:
-        - python
-        - {{ pin_subpackage('pyqt5-sip', exact=True) }}
-        - {{ pin_subpackage('pyqt-impl', exact=True) }}
-        - {{ pin_subpackage('pyqtwebengine', exact=True) }}
-        - {{ pin_subpackage('pyqtchart', exact=True) }}
     test:
-      files:
-        - pyqt_test.py
       imports:
-        - PyQt5.sip
         - PyQt5.QtChart
-        - PyQt5.QtCore
-        - PyQt5.QtGui
-        - PyQt5.QtHelp
-        - PyQt5.QtMultimedia
-        - PyQt5.QtMultimediaWidgets
-        - PyQt5.QtNetwork
-        - PyQt5.QtOpenGL
-        - PyQt5.QtPrintSupport
-        - PyQt5.QtQml
-        - PyQt5.QtQuick
-        - PyQt5.QtSvg
-        - PyQt5.QtTest
-        - PyQt5.QtWebChannel
-        - PyQt5.QtWebEngine
-        - PyQt5.QtWebEngineCore
-        - PyQt5.QtWebEngineWidgets
-        - PyQt5.QtWebSockets
-        - PyQt5.QtWidgets
-        - PyQt5.QtXml
-        - PyQt5.QtXmlPatterns
-      commands:
-        - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'python pyqt_test.py'  # [linux and build_platform == target_platform]
-        - pyuic5 --version  # [not win]
-        - conda list  # [not win]
 
 about:
-  home: http://www.riverbankcomputing.co.uk/software/pyqt
-  license: LicenseRef-Commercial or GPL-2.0-only or GPL-3.0-only
+  home: https://www.riverbankcomputing.com/software/pyqt/
+  license: GPL-3.0-only
   license_family: GPL
-  license_file: pyqt5/LICENSE
-  summary: 'Python binding of the cross-platform GUI toolkit Qt.'
+  license_file: pyqt/LICENSE
+  summary: 'Python bindings for the Qt cross platform application toolkit'
+  # The remaining entries in this section are optional, but recommended.
   description: |
-    "PyQt is a set of Python v2 and v3 bindings for The Qt Company's Qt
-    application framework and runs on all platforms supported by Qt including
-    Windows, MacOS/X and Linux. PyQt5 supports Qt v5. PyQt4 supports Qt v4 and
-    will build against Qt v5. The bindings are implemented as a set of Python
-    modules and contain over 1,000 classes."
-  doc_url: https://www.riverbankcomputing.com/software/pyqt/
-  dev_url: https://github.com/pyqt
+    PyQt5 is a comprehensive set of Python bindings for Qt v5.
+    It is implemented as more than 35 extension modules and enables Python
+    to be used as an alternative application development language to C++ on
+    all supported platforms including iOS and Android.
+  doc_url: https://www.riverbankcomputing.com/static/Docs/PyQt5/
+  dev_url: https://www.riverbankcomputing.com/mailman/listinfo/pyqt/
 
 extra:
```

Comment:
could add
```
  feedstock-name: pyqt
```
(below maintainers) so that the REAME doesn't pick up "pyqt-split".